packages/ui/src/components/va-stepper/VaStepper.demo.vue

```diff
@@ -94,4 +167,11 @@ const stepsWithCustomIcons = [
   { label: 'Four', icon: 'grade' },
   { label: 'Five', icon: 'list' },
 ]
+
+const stepsWithNextAction = ref([
+  { label: 'One' },
+  { label: 'Two', beforeSave: (step) => { step.hasError = true } },
```

Comment:
I wouldn't cal it save. No saving is happening.

Maybe something like `onOut`?